cmd/karavictl/cmd/role.go

```diff
@@ -147,6 +148,11 @@ var GetPowerMaxEndpoint = func(storageSystemDetails System) string {
 	return storageSystemDetails.Endpoint
 }
 
+// GetPowerScaleEndpoint returns the endpoint URL for a PowerMax system
```

Comment:
Comment should be endpoint URL for PowerScale not PowerMax?

Reply:
Fixed.